controllers/openstackcontrolplane_controller.go

```diff
@@ -143,7 +143,7 @@ func (r *OpenStackControlPlaneReconciler) Reconcile(ctx context.Context, req ctr
 			Role:                controlplane.Role,
 			HostCount:           controlplane.Count,
 			VIP:                 true,
-			AddToPredictableIPs: false,
+			AddToPredictableIPs: true,
```

Comment:
Oh, have we had this wrong all along?  Or is this necessary for these new changes somehow?

Reply:
I realized that /etc/hosts entries from the overcloud deployment were wrong. instead of the overcloud VIPs we added the openstackclient ips ...